Assets/Scripts/Gameplay/GameState/BossRoomState.cs

```diff
@@ -93,7 +97,8 @@ void OnServerLoadComplete(ulong clientId, string sceneName, LoadSceneMode loadSc
             }
 
             // reset win state
```

Comment:
since the static instance is set on Awake here and destroyed in PostGameState's OnDestroy, I suppose this is no longer required?